views/healthcheck/index.js

```diff
@@ -13,5 +15,22 @@ module.exports = view.extend({
         back: '/profile',
         config: config,
         package: packageJSON
+    },
+    computed: {
+      options: function() {
+        return i18n.supportedLangs();
+      },
+      currentLang: function() {
+        return i18n.locale;
+      }
+    },
+    methods: {
+      langmap: function(locale) {
+        return langmap[locale].nativeName;
+      },
+      selectedLang: function(locale) {
+        i18n.setLocale(locale, true);
```

Comment:
you can just use `i18n.setLocale(this.currentLang, true)` if you set `v-model="currentLang"` on the select above